packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -266,25 +264,36 @@ contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
     override
     nonReentrant
   {
+    // called by aragon court
     require(_ruling <= DISPUTES_POSSIBLE_OUTCOMES, "invalid ruling");
```

Comment:
This is where my confusion about `DISPUTES_POSSIBLE_OUTCOMES` is coming from. If I'm understanding this line correctly, it would require that any `_ruling` be 5 or lower, which would mean rulings of 6 and 7 would revert.

Reply:
Thank you. This is incorrect.